platform-sdk/swirlds-config-extensions/src/main/java/com/swirlds/config/extensions/sources/SimpleConfigSource.java

```diff
@@ -182,14 +182,16 @@ public SimpleConfigSource withValue(final String propertyName, final Long value)
      * @param propertyName name of the peoprty
      * @param value        default value
      */
-    public SimpleConfigSource withValue(final String propertyName, final Boolean value) {
+    @NonNull
+    public SimpleConfigSource withValue(@NonNull final String propertyName, @NonNull final Boolean value) {
         setValue(propertyName, value, v -> Boolean.toString(v));
         return this;
     }
 
-    private <T> void setValue(final String propertyName, final T value, Function<T, String> converter) {
-        CommonUtils.throwArgBlank(propertyName, "propertyName");
```

Comment:
We have that functionality in base. please use that and not change it to notNull